verticapy/tests_new/performance/vertica/test_conn_simple.py

```diff
@@ -0,0 +1,105 @@
+"""
+Copyright  (c)  2018-2024 Open Text  or  one  of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+import pytest
+import logging
+
+import verticapy as vp
+import vertica_python
+
+from verticapy.performance.vertica import QueryProfiler
+
+from io import StringIO
+
+
+def test_profile_simple(vp_connect_get_name, raw_client_cursor):
+    """Create a query profiler and run the steps on a simple query"""
+
+    table_name = "test_profile_simple"
+    setup_dummy_table(raw_client_cursor, table_name)
+    request = f"SELECT count(*) from {table_name};"
+    qp = QueryProfiler(request)
+    check_version(qp)
+    check_request(qp, "SELECT COUNT(*)")
+
+
+def setup_dummy_table(raw_client_cursor, table_name):
```

Comment:
@jslaunwhite-microfocus - we run test in parallel. It would be better to create unique schema to avoid conflicts. You can use existing fixture load_test_schema that can server this purpose.

Reply:
The statement was removed.